### DISCUSSION TITLE
Why does starting Ente on iOS or Android always show "Encrypting backup..." and a loading spinner for long durations

### DISCUSSION THREAD

**Discussion by pczern** [ente-io/ente#2529] [Q&A] [answered]
On my android device the spinner appears for 2mins after start and on my iOS device for a longer time and only disappears as it seems first after viewing images.

**Comment by maffinca69** [comment]
The same thing. The backup copy is not saved and in fact nothing works
@pczern Have you managed to fix it?

**Comment by maffinca69** [comment, marked as answer]
@pczern I fixed! See https://github.com/ente-io/ente/issues/2601#issuecomment-2264893840
